acceptance/Gemfile

```diff
@@ -18,6 +18,7 @@ gem "beaker-abs", *location_for(ENV['BEAKER_ABS_VERSION'] || "~> 0.2")
 gem 'rake'
 gem 'scooter', '~> 3.2.17'
 gem 'pcp-client', '~> 0.4'
+gem 'hocon'
```

Comment:
Why are you including the gem rather than using the hocon helper we already have for tests? https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/helpers/hocon_helpers.rb#L56